samples/quickstart.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+from google.cloud.bigquery import reservation_v1
```

Comment:
⚠️  Using the `google.cloud.bigquery` namespace is going to fail spectacularly if you try to install `google-cloud-bigquery` at the same time as this package. Pip installs things by just overwriting directories, so the last one to be installed will "win" ⚠️ 

Nit: Should this be inside the quickstart region tag? Seems like we want to show imports in our samples.

Reply:
I looked into this a little more. Sounds like this is pretty much fixed (even for conda) on Python 3. https://github.com/conda/conda-build/issues/649#issuecomment-230586593